src/pat/recurrence/recurrence.js

```diff
@@ -660,7 +660,7 @@ const RecurrenceInput = function (conf, textarea) {
     var now_date = new Date().toISOString().substring(0, 10);
     var index, i;
 
-    for (i = 0; i < 7; i++) {
+    for (let i = 0; i < 7; i++) {
```

Comment:
the initialization of `var i` above should be removed.

Reply:
I've removed every obsolete declarations.